test/e2e/config/docker.yaml

```diff
@@ -323,11 +323,11 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.26.0"
-  KUBERNETES_VERSION: "v1.26.0"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.25.3"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.26.0"
-  ETCD_VERSION_UPGRADE_TO: "3.5.6-0"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.27.0"
+  KUBERNETES_VERSION: "v1.27.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.26.0"
```

Comment:
Should this be the [latest](https://kubernetes.io/releases/) of `1.26.3` like previous?